src/servers/graphql-old-server.ts

```diff
@@ -384,7 +361,7 @@ const resolvers = {
         let feeSatAmount: Satoshis | ApplicationError
         if (amount && amount > 0) {
           feeSatAmount = await getNoAmountLightningFee({
-            walletPublicId: wallet.user.walletPublicId,
+            walletId: wallet.user.id,
```

Comment:
Should this be `wallet.user.walletId` after the migration?

Reply:
I think this should be `user.id` here. where are the other place where we still access `user.walletId`? 

there are some placed that I have replaced here: https://github.com/GaloyMoney/galoy/pull/808/commits/e6d1022f74ca432b0711aae63bfe24b85c98af21

Reply:
Sorry I was looking at the wrong branch locally 🙈  ... approved ✅